services/galley/test/integration/API/Teams.hs

```diff
@@ -1053,6 +1063,107 @@ testDeleteBindingTeamSingleMember = do
   -- Let's clean the queue, just in case
   ensureQueueEmpty
 
+testDeleteTeamVerificationCodeSuccess :: TestM ()
+testDeleteTeamVerificationCodeSuccess = do
+  g <- view tsGalley
+  (owner, tid) <- Util.createBindingTeam'
+  let Just email = U.userEmail owner
+  setTeamSndFactorPasswordChallenge tid Public.TeamFeatureEnabled
+  generateVerificationCode $ Public.SendVerificationCode Public.DeleteTeam email
+  code <- getVerificationCode (U.userId owner) Public.DeleteTeam
+  delete
+    ( g
+        . paths ["teams", toByteString' tid]
+        . zUser (U.userId owner)
+        . zConn "conn"
+        . json (newTeamDeleteDataWithCode (Just Util.defPassword) (Just code))
+    )
+    !!! do
+      const 202 === statusCode
+  tryAssertQueue 10 "team delete, should be there" tDelete
+  ensureQueueEmpty
```

Comment:
```suggestion
  assertQueueEmpty
```

If that fails, there is something in the queue that we may want to check for as well.  Same for the following test cases.

(Can be done in a separate mini-PR, or not at all.)